doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -182,51 +180,44 @@ You can download latest Service OS kernel from
                 linux  /boot/acrn/org.clearlinux.iot-lts2018-sos.4.19.78-98  pci_devices_ignore=(0:18:1) console=tty0 console=ttyS0 root=PARTUUID=<UUID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1 i915.enable_guc=0 hvlog=2M@0x1FE00000
         }
 
-   .. note::
-        You need to adjust this to use your partition UUID (``PARTUUID``) for
-        the ``root=`` parameter (or use the device node directly).
+     .. note::
+          Adjust this to use your partition UUID (``PARTUUID``) for the ``root=`` parameter (or use the device node directly).
 
-   .. note::
-        You will also need to adjust the kernel name if you used a different
-        RPM file as the source of your Service OS kernel.
+          Adjust the kernel name if you used a different RPM file as the source of your Service VM kernel.
 
-   .. note::
-        The command line for the kernel in /etc/grub.d/40_custom should be all
-        as a single line, not as multiple lines. Otherwise the kernel will fail to boot
+          The command line for the kernel in ``/etc/grub.d/40_custom``
+          should be entered as a single line, not as multiple lines. Otherwise, the kernel will fail to boot.
 
-   * Modify the ``/etc/default/grub`` file to make the grub menu visible when booting
-     and make it load the Service OS kernel by default.
-     There are a few lines to be modified, as shown below.
+   * Modify the ``/etc/default/grub`` file to make the grub menu visible
+     when booting and make it load the Service VM kernel by default.
+     Modify the lines shown below:
 
      .. code-block:: none
 
         GRUB_DEFAULT=ubuntu-service-vm
         #GRUB_TIMEOUT_STYLE=hidden
         GRUB_TIMEOUT=3
 
-   * Update Grub on your system
+   * Update Grub on your system:
 
      .. code-block:: none
 
         sudo update-grub
 
-#. Reboot the system
+#. Reboot the system.
 
-   Reboot system. You should see the Grub menu with the new "ACRN ubuntu SOS"
-   entry. Select it and proceed to booting the platform. The system will start
-   the Ubuntu Desktop and you can now log in (as before).
+   Reboot the system. You should see the Grub menu with the new "ACRN ubuntu SOS" entry. Select it and proceed to booting the platform. The system will start the Ubuntu Desktop and you can now log in (as before).
```

Comment:
Let's change it here too actually (and you'll need to make a similar adjustment to the line above that shows the `menuentry`:
```suggestion
   Reboot the system. You should see the Grub menu with the new "ACRN Ubuntu Service VM" entry. Select it and proceed to booting the platform. The system will start the Ubuntu Desktop and you can now log in (as before).
```